server/appServices.js

```diff
@@ -17,9 +18,24 @@ const KEYCLOAK_SECRET_SUFFIX = '-install-config';
 const DATASYNC_CONFIGMAP_SUFFIX = '-data-sync-binding';
 const MOBILE_SECURITY_SUFFIX = '-security';
 
+const { MAX_RETRIES = 360, RETRY_TIMEOUT = 5000 } = process.env;
```

Comment:
This will make the MDC retry for 30 minutes. Shouldn't be about 5 minutes instead?

Reply:
you're right. thanks!